google/resource_compute_region_instance_group_manager.go

```diff
@@ -257,17 +257,17 @@ func waitForInstancesRefreshFunc(f getInstanceManagerFunc, d *schema.ResourceDat
 			log.Printf("[WARNING] Error in fetching manager while waiting for instances to come up: %s\n", err)
 			return nil, "error", err
 		}
-		if creatingCount := m.CurrentActions.Creating + m.CurrentActions.CreatingWithoutRetries; creatingCount > 0 {
-			return creatingCount, "creating", nil
+		if done := m.CurrentActions.None; done < m.TargetSize {
```

Comment:
_slaps forehead_

I think I remember writing this code, so thanks for catching my mistake there - obviously the instances shouldn't report ready if they're `FAILED`.